web/source/keymanweb.ts

```diff
@@ -1860,45 +1365,7 @@ if(!window['keyman']['initialized']) {
      * 
      */  
     keymanweb['removeKeyboards'] = function(x) {
-      if(arguments.length == 0) {
-        return false;
-      }
-
-      var i,j,ss=keymanweb._KeyboardStubs;
-      var success = true, activeRemoved = false, anyRemoved = false;;
-
-      for(i=0; i<arguments.length; i++) {           
-        for(j=ss.length-1; j>=0; j--) {
-          if('Keyboard_'+arguments[i] == ss[j]['KI'] && ss.length > 1) {                 
-            if('Keyboard_'+arguments[i] == keymanweb['getActiveKeyboard']()) {
-              activeRemoved = true;
-            }
-
-            anyRemoved = true;
-            ss.splice(j,1);
-            break;
-          }
-        }
-
-        if(j < 0) {
-          success = false;
-        }
-      } 
-
-      if(activeRemoved) {
-        // Always reset to the first remaining keyboard
-        keymanweb._SetActiveKeyboard(ss[0]['KI'],ss[0]['KLC'],true);
-        // This is likely to be triggered by a UI call of some sort, and we need to treat
-        // this call as such to properly maintain the globalKeyboard setting.
-        keymanweb._JustActivatedKeymanWebUI = 1;
-      }
-
-      if(anyRemoved) {
-        // Update the UI keyboard menu
-        keymanweb.doKeyboardUnregistered();
-      }
-        
-      return success;
+      return keymanweb.keyboardManager.removeKeyboards(x);
```

Comment:
`removeKeyboards` originally supported multiple arguments. This doesn't appear to.

Reply:
How so?  I just pulled up both versions side by side, and the code is nigh-identical - just transplanted into kmwkeyboards.ts.  The function here is designed to forward the original API call to its new location.